src/main/groovy/org/mockito/release/internal/gradle/ReleaseConfigurationPlugin.java

```diff
@@ -52,17 +51,30 @@ public void apply(Project project) {
             }
 
             configuration.setPreviousReleaseVersion(info.getPreviousVersion());
+
+            TaskMaker.task(project, INIT_CONFIG_FILE_TASK, InitConfigFileTask.class, new Action<InitConfigFileTask>() {
+                @Override
+                public void execute(InitConfigFileTask t) {
+                    t.setDescription("Creates config file if it doesn't exist");
+                    t.setConfigFile(configFile);
+
+                    project.getTasks().getByName(BootstrapPlugin.INIT_SHIPKIT_TASK).dependsOn(t);
+                }
+            });
+
         } else {
             //not root project, get extension from root project
             configuration = project.getRootProject().getPlugins().apply(ReleaseConfigurationPlugin.class).getConfiguration();
         }
     }
 
-    private void loadShipKitConfigFile(Project rootProject) {
-        final File configFile = rootProject.file(CONFIG_FILE_RELATIVE_PATH);
+    private void loadConfigFromFile(Project rootProject, File configFile) {
         if (!configFile.exists()) {
-            createShipKitConfigFile(configFile);
-            throw new GradleException("Config file created at " + configFile.getAbsolutePath() + ". Please configure it and rerun the task.");
+            // sets some defaults so that they can't be used to run any task (except for bootstrap ones)
+            // but also configuration doesn't fail when running Shipkit for the first time
```

Comment:
Don't you think it's better to keep the exemplary correct values, especially for repository? It tells you immediately what format is the property value ("mockito/mockito" or "https://github.com/mockito/mockito"?). The token provided here is not correct btw.